File: JetMovie/JetMovie/src/app/movies/models/movie-request.ts
export enum SortOptions {
    None,
    ByDate,
    ByDateDesc,
    ByTitle,
    ByTitleDesc,
    ByCountry,
    ByCountryDesc
}
export class MovieRequest {
    public Title: string;
    public Year: number;
    public Genre: string;
    public Description: string;
    public Country: string;
    public Actors: string;
    public SortBy: SortOptions;
    public Page: number;
    public PageSize: number;
}

File: JetMovie/JetMovie/Models/ViewModels/MovieRequest.cs
namespace JetMovie.Models.ViewModels
{
    public class MovieRequest
    {
        public string Title { get; set; }
        public int Year { get; set; }
        public string Genre { get; set; }
        public string Description { get; set; }
        public string Country { get; set; }
        public string Actors { get; set; }
        public SortOptions SortBy { get; set; }
        public int Page { get; set; }
        public int PageSize { get; set; }
    }
}